packages/airnode-deployer/src/utils/logger.ts

```diff
@@ -84,3 +90,21 @@ export function debugMode(mode: boolean) {
 export function inDebugMode() {
   return debugModeFlag;
 }
+
+export function setSecret(secret: string) {
+  secrets.push(secret);
+}
+
+export function replaceSecrets(input: string, secrets: string[]) {
+  let output = input;
+  secrets.forEach((secret) => (output = output.replace(secret, '*'.repeat(secret.length))));
+
+  return output;
+}
+
+export function setLogsDirectory(path: string) {
+  logsDirectory = path.endsWith('/') ? path.slice(0, -1) : path;
+  if (!fs.existsSync(logsDirectory)) fs.mkdirSync(logsDirectory, { recursive: true });
```

Comment:
Hmm, I thought that since [`fs.exists` is being deprecated](https://nodejs.org/api/fs.html#fsexistspath-callback) `fs.existsSync` will be as well but it's not. Either way, I started using [`fs.accessSync`](https://nodejs.org/api/fs.html#fsaccesssyncpath-mode) instead. Can you do the same to be consistent?